### ISSUE TITLE
Android: Failure detail text missing engine versions

### ISSUE THREAD

**Issue by dbjorge** [microsoft/accessibility-insights-web#4490]
## Describe the bug

The end of the text you get when using "File issue" or "Copy failure details" from Accessibility Insights for Android currently reads:

```
Environment: 

====

This accessibility issue was found using Accessibility Insights for Android 2021.518.3 (axe-android), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.
```

This text should mention both of the rules engines we use (not just axe-android), and it should include version numbers for both.

It should also not have a confusing empty `Environment:` block. Either the `\n====` should be omitted if the footer text is intended to represent the environment, or the `Environment:` line should be omitted if doing a visual break and a separate footer is intended, not both.

## CodePen repro example

n/a

## Expected behavior

Footer block should concisely identify engine version numbers

## Screenshots

n/a

## Context (please complete the following information)

- OS Name & Version: Repros on Ubuntu 20.04 and Windows 21H1 19043.1110
- AI-Web Version & Environment: AI-Android 2021.518.3 (current prod) and 2021.714.15 (current insider)
- Browser Version: n/a
- Target Page: any (Sunflower demo app)

## Are you willing to submit a PR?

yes

## Did you search for similar existing issues?

yes

## Additional context

n/a

**Comment by peterdur** [comment]
Quick notes:

- not having the axe-android version may be intentional per #2757 
- in Web, environment is the browser and version; for Android, do we want this to be the device/app as in the report?

**Comment by ferBonnin** [comment]
marking as needs investigation and assigning to @juchampa to define requirements on the axe-android version and if we need environment details. Since this might be intentional, we might want to check why we don't have the axe-android version

**Comment by ferBonnin** [comment]
triaged with Peter
* agreed to add the axe-android version
* in Web, environment is the browser and version; for Android, do we want this to be the device/app as in the report: yes

**Comment by DaveTryon** [comment]
We won't be fixing this
